### PR TITLE
Added One to many and Mockito

### PR DESCRIPTION
**Pull request by mounishdodda**
Added One to many and Mockito

### CODE REVIEW

**Comment by phanindraduvvuri** [comment]
Did you run the code after this.

**Comment by phanindraduvvuri** [comment]
after adding relationship in CourseEntity.

File: src/main/java/com/cg/ams/entity/CourseEntity.java [hunk 1]
```diff
@@ -4,20 +4,36 @@
...
+
+	@Id
     @GeneratedValue
     private long id;
     private String name;
     private String description;
+    
+    @OneToMany(cascade=CascadeType.ALL)
+	@JoinColumn(name="subjectId")
+	private List<SubjectEntity> subject;
```
Comment (phanindraduvvuri) [inline src/main/java/com/cg/ams/entity/CourseEntity.java:37]: They did not gave any bidirectional relationship from course, right?

Comment (mounishdodda) [reply src/main/java/com/cg/ams/entity/CourseEntity.java:37]: Yes Did not specifies Bi directional but Sir said to add . If it is not necessary No need

**Review by phanindraduvvuri — APPROVED** [review]

**Pull request merged**